docs/en/basic/Command-Line-Interface.md

```diff
@@ -1061,7 +1061,7 @@ $ ./bin/alluxio fs unsetTtl /data/yesterday/data-not-yet-analyzed
 
 ## Table Operations
 
-```
+```console
 ./bin/alluxio table
```

Comment:
can you add a `$` at the beginning of this line since we're changing the block to `console` type

```suggestion
$ ./bin/alluxio table
```

Reply:
done